src/main/java/ru/itmo/sd/deadliner2bot/ui/commands/Commands.java

```diff
@@ -0,0 +1,57 @@
+package ru.itmo.sd.deadliner2bot.ui.commands;
+
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
+import org.springframework.stereotype.Component;
+import ru.itmo.sd.deadliner2bot.model.ChatStateEnum;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.text.MessageFormat;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Component
+public class Commands {
+
+    private Properties commandsProperties;
+
+    @PostConstruct
+    public void postConstruct() throws IOException {
+        Resource resource = new ClassPathResource("messages/state-commands.properties");
+        commandsProperties = PropertiesLoaderUtils.loadProperties(resource);
+    }
+
+    public Map<String, CommandInfo> loadAllCommandsInfo(ChatStateEnum chatStateEnum, List<String> commandKeys) {
+        Map<String, CommandInfo> result = new HashMap<>();
+        commandKeys.forEach(commandKey -> result.put(commandKey, getCommandInfoForChatState(chatStateEnum, commandKey)));
```

Comment:
May be check here if given commandKey is actual key in map? otherwise we will put <key, null> pair in result map.